### PR TITLE
refactor: swaps ui with leather-styles

### LINKED ISSUE #4185: Update swap UI w/ new design system

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Leather — download [extension builds](https://github.com/leather-wallet/extension/actions/runs/6113652435).

This PR refactors the swaps UI to use the new brand/design system. It also changes the summary tabs to use routes, cc @pete-watters!

Screen Shot 2023-09-07 at 12 36 02 PM

Screen Shot 2023-09-07 at 12 36 10 PM

Screen Shot 2023-09-07 at 1 06 16 PM

### CODE REVIEW

File: src/app/pages/swap/swap-summary/swap-summary.tsx [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+import WaxSeal from '@assets/illustrations/wax-seal.png';
+import { useClipboard } from '@stacks/ui';
 import { useFormikContext } from 'formik';
+import { HStack, styled } from 'leather-styles/jsx';
 
 import { logger } from '@shared/logger';
 import { isUndefined } from '@shared/utils';
 
 import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
 // import { useExplorerLink } from '@app/common/hooks/use-explorer-link';
```
Comment (pete-watters) [inline src/app/pages/swap/swap-summary/swap-summary.tsx:13]: Can you please delete this line now too? I know it was already like that

`// import { useExplorerLink } f..`

File: src/app/pages/swap/swap-summary/swap-summary.tsx [hunk 2]
```diff
@@ -58,30 +58,30 @@ export function SwapSummary() {
...
+        </styled.h3>
         <SwapAssetsPair />
-        <SwapSummaryTabs details={<SwapDetails />} status={<SwapStatus />} />
+        <SwapSummaryTabs>
+          <Outlet />
+        </SwapSummaryTabs>
       </SwapContentLayout>
       <SwapFooterLayout>
-        <Stack isInline spacing="base" width="100%">
+        <HStack gap="space.04" width="100%">
```
Comment (pete-watters) [inline src/app/pages/swap/swap-summary/swap-summary.tsx:77]: Nice work here, I was trying to figure out how to replace `<Stack isInline`, I'll use `<HStack` 👍

**Review by pete-watters — APPROVED** [review]
Great work here @fbwoolf 👍 your changes to `spacing` and the font styles are helping me also when refactoring away `stacks/ui`

File: src/app/pages/swap/swap-summary/swap-summary.tsx (hunk 1 above)
Comment (fbwoolf) [reply src/app/pages/swap/swap-summary/swap-summary.tsx:13]: These comments are still being left for devs working on other parts of this feature. I believe they can be removed soon with the alex PR I'm helping out with atm.

**Review by alter-eggo — APPROVED** [review]

**Comment by mica000** [comment]
Looking good @fbwoolf!

---------

**ROW HEIGHT**
What do you tink of making every row a fixed height? it will prevent changing the size according to the content. In this case, we would follow the heigh of the row with 2 lines
Screenshot 2023-09-08 at 10 32 55

---------

**LINK STYLE**
What do you think of using the link style on this component?
Screenshot 2023-09-08 at 10 34 46
https://www.figma.com/file/2MLHeIeL6XPVi3Tc2DfFCr/%E2%9D%96-Leather-%E2%80%93-Design-System?type=design&node-id=5787-90549&mode=design&t=tn88fGYw436PHNQJ-4

---------


**HEADER AND SPACING AT THE TOP**
I believe the header title will be solved on the Container's ticket, but wondering about the bigger spacing that seems to be on top - is this also part of that PR or it's related to the swap wrapper? 
Screenshot 2023-09-08 at 10 32 36

**Comment by markmhendrickson** [comment]
@mica000 since this PR was merged, it may be best to migrate your above comment into separate issue(s) for open tracking?

**Comment by fbwoolf** [comment]
> @mica000 since this PR was merged, it may be best to migrate your above comment into separate issue(s) for open tracking?

I can make these updates in the alex integration PR bc they are pretty minor.

**Comment by fbwoolf** [comment]
> Looking good @fbwoolf!
> 
> **ROW HEIGHT** What do you tink of making every row a fixed height? it will prevent changing the size according to the content. In this case, we would follow the heigh of the row with 2 lines

Yep, will do.

> **LINK STYLE** What do you think of using the link style on this component?

Yep, I can change.

> **HEADER AND SPACING AT THE TOP** I believe the header title will be solved on the Container's ticket, but wondering about the bigger spacing that seems to be on top - is this also part of that PR or it's related to the swap wrapper?

I think I did this bc the header title is off from where it is on the design, but the space b/w it and the label over the selector is what I am showing? If you want it moved up, np.

**Pull request merged**